File: src/main/java/de/maggiwuerze/xdccloader/persistence/ServerRepository.java
package de.maggiwuerze.xdccloader.persistence;

import de.maggiwuerze.xdccloader.model.entity.Server;
import java.util.List;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface ServerRepository extends CrudRepository<Server, Long> {

	List<Server> findAll();
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/persistence/ServerRepository.kt
package de.maggiwuerze.xdccwebloader.persistence

import de.maggiwuerze.xdccwebloader.persistence.entity.Server
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface ServerRepository : CrudRepository<Server, UUID> {
    override fun findAll(): List<Server>
    fun findByName(name: String): Server?
}